scripts/install_from_src_macos

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -euo pipefail
+
+clean_slate()
+{
+	echo "                      =============================================="
+	echo "                      If you are within a conda env please deactivate it and come back"
```

Comment:
We don't want to support install from source on conda environment? 
As per manual testing the installation works fine on conda environment as well. 

The script should support installation on pip/pyenv and conda environments.